client/src/pages/board/KanbanBoard.jsx

```diff
@@ -156,21 +155,19 @@ export default function KanbanBoard() {
     }
 
     if (type === 'column') {
-      await dispatch({
-        type: 'MOVE_COL',
-        fromIndex: source.index,
-        toIndex: destination.index,
-      });
+      boardActions.moveColumn(source.index, destination.index, dispatch);
+      await Board.saveData(data.id, {columns: data.columnOrder});
```

Comment:
In my hotfix that has been merged onto dev, I relocated this api call into the reducers of boardContext.js. I was thinking that this is better than leaving it in the frontend component since it's easier to see how the react state is sent to the backend.

Reply:
Yeah, you're completely right about that. My updateColumns function is left in the reducers, so this one doesnt need to be there.